docs/dataset/advanced-usage/extract-metadata-text.md

```diff
@@ -0,0 +1,274 @@
+---
+icon: kolena/comparison-16
+---
+
+# :kolena-comparison-20: Text Metadata Hydration
```

Comment:
Let's keep the terminology the same here.  How about instead of `Hydration` we use `Extraction`?